### PR TITLE
Meta outlier classifier

### PR DESCRIPTION
**Pull request by koaning**
This turns outlier detectors into classifiers.

### CODE REVIEW

**Comment by koaning** [comment]
@MBrouns some of the classifier tests totally break here, but I think this one will resolve https://github.com/koaning/scikit-lego/issues/323.

File: sklego/meta/outlier_classifier.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    """
+    Morphs an estimator that performs outlier detection into a classifier.
+    When an outlier is detected it will output 1 and 0 otherwise.
+    This way you can use familiar metrics again and this allows you
+    to consider outlier models as a fraud detector.
+    """
+    def __init__(self, model):
+        self.model = model
+
+    def _is_outlier_model(self):
```
Comment (MBrouns) [inline sklego/meta/outlier_classifier.py:19]: Would this be solved in a nicer way by implementing another meta model, like the `ProbabilisticClassifier`


```class OutlierModelMeta(type):
    def __instancecheck__(self, other):
        return isinstance(self.model, OutlierMixin)

class OutlierModel(metaclass=OutlierModelMeta):
    pass
```

Comment (koaning) [reply sklego/meta/outlier_classifier.py:20]: Valid. Made the changes.

**Comment by koaning** [comment]
@MBrouns should be fixed. I added your request, removed the notebook and added one extra test.

**Pull request merged**